tox.ini

```diff
@@ -1,13 +1,15 @@
 [tox]
-envlist = py27,py33,py34,pypy,docs,pep8,py3pep8,gendoc
+envlist = {py27,py33,py34,pypy},docs,pep8,py3pep8,gendoc,coverage-report
 
 [testenv]
+coverage_pytest_cmd = coverage run --parallel -m pytest --capture=no --strict
+hypothesis_args = --run-hypothesis
 deps =
     coverage
     pytest
 commands =
-    coverage run -m pytest --capture=no --strict {posargs}
-    coverage report -m --fail-under 100
+     {py27,py34,py35,pypy}: {[testenv]coverage_pytest_cmd} {[testenv]hypothesis_args}
```

Comment:
This is the first time in this repo `py35` has shown up. Can you check if we should include it in `envlist` above and `.travis.yml` as well?